package college2.student;
public class Students{
    String name;
    int rollNo;
    public Students(String name,int rollNo){
        this.name=name;
        this.rollNo=rollNo;
    }
    public void displayStudents(){
        System.out.println("Student Name: "+name);
        System.out.println("Roll Number: "+rollNo);
    }
}
